src/cmd/sql_cmd.h

```diff
@@ -607,6 +610,16 @@ void HandleCmd(const hybridse::node::CmdPlanNode* cmd_node) {
             }
             break;
         }
+        case hybridse::node::kCmdShowSessionVariables: {
+            for (auto& pair : session_variables) {
+                std::cout << pair.first << "=" << pair.second << std::endl;
```

Comment:
Print the variables like MySQL does, or simply with headers.

```
mysql> show variables like "window%";
+------------------------------+-------+
| Variable_name                | Value |
+------------------------------+-------+
| windowing_use_high_precision | ON    |
+------------------------------+-------+
1 row in set (0.00 sec)
```

Reply:
If we move session variables into sql router, we will get `ResultSet`, and we can print `ResultSet` like this. I think it's no need to make the map print look like ResultSet.

Reply:
Or we can convert the session variable map to ResultSet? We can reuse it later.

Reply:
We use `PrintItemTable` to print it temporarily.